From 7b8e9ea4a17ccaf3418c48294ae13b784d19cb63 Mon Sep 17 00:00:00 2001
From: Alexander Larsson <alexl@redhat.com>
Date: Mon, 20 Aug 2018 14:56:00 +0200
Subject: [PATCH] dbus xml: Delete -- from docs

Apparently -- is not valid XML, so a nonbreakable space was added, but
that breaks gdbus-codegen, so lets just drop the dashes totally.

Closes: #1988
Approved by: alexlarsson
---
 data/org.freedesktop.portal.Flatpak.xml | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/data/org.freedesktop.portal.Flatpak.xml b/data/org.freedesktop.portal.Flatpak.xml
index c3afdc2c..b7bbed29 100644
--- a/data/org.freedesktop.portal.Flatpak.xml
+++ b/data/org.freedesktop.portal.Flatpak.xml
@@ -67,13 +67,13 @@
            <varlistentry>
              <term>4</term>
              <listitem><para>
-               Spawn in a sandbox (equivalent of the -‍-sandbox option of flatpak run).
+               Spawn in a sandbox (equivalent of the sandbox option of flatpak run).
              </para></listitem>
            </varlistentry>
            <varlistentry>
              <term>8</term>
              <listitem><para>
-               Spawn without network (equivalent of the -‍-unshare=network option of flatpak run).
+               Spawn without network (equivalent of the unshare=network option of flatpak run).
              </para></listitem>
            </varlistentry>
          </variablelist>
